README.md

```diff
@@ -171,3 +434,19 @@ $ cat > templates/xml/my_mentoring_scenario.xml
 
 Restart the workbench to take the new scenarios into account.
 
+Upgrading from Version 1
+------------------------
+
+To upgrade a course from the earlier version of this XBlock, run the following
+command on a system with edx-platform and xblock-mentoring installed:
+
+```bash
+$ SERVICE_VARIANT=cms DJANGO_SETTINGS_MODULE="cms.envs.devstack" python -m mentoring.v1.upgrade "Org/Course/Run"
```

Comment:
@antoviaque It should work fine on either modulestore but I have only tested one. I will test both locally.

In this case, I think all that's happened is it didn't actually find the course and it's giving the wrong error message because modulestore doesn't actually throw an error when you request a non-existent course. Can you check that you correctly quoted the course identifier in the shell ("course-v1:HarvardX+GSE1.1x+3T2014")?